spec/integration/api_resource_evaluations_controller_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'swagger_helper'
+require 'rails_helper'
+
+describe ApiResourceEvaluationsController do
+  let(:user) { create(:user) }
+
+  before do
+    sign_in user
+  end
+
+  path '/api/resources/{resource_id}/resource_evaluation' do
+    put 'Creates or modifies the evaluation of a resource' do
+      tags 'ResourceEvaluations'
```

Comment:
The tag is used by the swagger documentation, son it's better if is human-readable, for example "Resource Evaluations"

Reply:
Ok. I will make the change :)

Reply:
It is already done :)